app/helpers/application_helper/button/container_perf.rb

```diff
@@ -1,8 +1,17 @@
 class ApplicationHelper::Button::ContainerPerf < ApplicationHelper::Button::Basic
   needs(:@record)
 
+  def initialize(view_context, view_binding, instance_data, props)
+    super
+    @entity = props[:options][:entity]
```

Comment:
This should probably check that @entity is not nil - to catch toolbars which don't provide the entity info.

Reply:
👍